### PR TITLE
[RAINCATCH-1330] Extend Passportauth to add developer story

### PR DESCRIPTION
**Pull request by JameelB**
## What
The current PassportAuth documentation provides little information for overall developer story. Extend the current PassportAuth documentation to provide more info on the developer story.

## Progress
- [x] Add PassportAuth developer story
- [x] Replace urls to core to point to server

## Additional Notes
Related JIRA - https://issues.jboss.org/browse/RAINCATCH-1330

### CODE REVIEW

File: docs/workforce-management-framework/topics/pro-passportauth-enablement.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
+[id='{context}pro-passportauth-enablement']
+= Enabling PassportAuth
+
+== Prerequisites
+. Ensure you have completed xref:getting-started[Getting Started]
+
+== Enabling PassportAuth on the {PROJECT_NAME} Applications
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-passportauth-enablement.adoc:10]: It clashes with enabled by default. Assume that person just started. This statement may be confusing

Comment (wtrocki) [reply docs/workforce-management-framework/topics/pro-passportauth-enablement.adoc:10]: just say if you enqbled keycloak previously

File: docs/workforce-management-framework/topics/pro-passportauth-implementation.adoc [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+=== Initialize a PassportAuth Service
+PassportAuth can be initialized in two different ways depending on what Passport.js strategy you would like to use.
+
+For more information on PassportAuth init function, see PassportAuth
+link:./../../api/{WFM-RC-Api-Version}{WFM-RC-Api-Passport-Auth}#init[init()] documentation.
+
+[discrete]
+==== Initialize PassportAuth to use the Local Strategy
+Passport.js link:https://github.com/jaredhanson/passport-local[local strategy] is used for authenticating with a username and password.
+In {PROJECT_NAME}, this strategy is used to authenticate users in the portal application.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-passportauth-implementation.adoc]: Worth add ' by default'

**Review by wtrocki — APPROVED** [review]
Best documentation in entire document. Really clear and simple. Ready to merge

File: docs/workforce-management-framework/topics/pro-passportauth-enablement.adoc (hunk 1 above)
Comment (JameelB) [reply docs/workforce-management-framework/topics/pro-passportauth-enablement.adoc:10]: I've moved the note for this to the top level of the document

**Pull request merged**